### DISCUSSION TITLE
Can’t take control over speed

### DISCUSSION THREAD

**Discussion by hoseth0711** [flybywiresim/aircraft#2392] [Help]
I flew in the a32nx today but on final it wouldn’t allow me to adjust speed manually even though I disabled AP and APP resulting in a very hard touch down

**Comment by donstim** [comment]
Autothrust is separate from the autopilot. Did you have autothrust on and what speed mode were you in (managed or selected)? you can disengage autothrust by either bringing the thrust levers back to idle, assigning a key or button to disengage autothrust, or pressing on the autothrust push button below the autopilot push button. The light should be extinguished on the autothrust push button when autothrust is not engaged.

I would recommend, however, that you keep autothrust engaged with the levers in the CL detent and in managed speed mode. This will give you the correct approach speed as you go through each flap setting and for full landing flaps on final. Flare at about 20-30 feet, bring the thrust levers to idle, and you should have a nice landing.

**Reply by hoseth0711** [reply]
Thank you so much! It was indeed the auto thrust
